eng/scripts/Create-ApiReview.ps1

```diff
@@ -0,0 +1,185 @@
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [array] $ArtifactList,
+  [Parameter(Mandatory=$True)]
+  [string] $ArtifactPath,
+  [Parameter(Mandatory=$True)]
+  [string] $APIViewUri,
+  [Parameter(Mandatory=$True)]
+  [string] $APIKey,
+  [Parameter(Mandatory=$True)]
+  [string] $SourceBranch,
+  [Parameter(Mandatory=$True)]
+  [string] $DefaultBranch,
+  [Parameter(Mandatory=$True)]
+  [string] $ConfigFileDir,
+  [Parameter(Mandatory=$True)]
+  [string] $buildId,
+  [Parameter(Mandatory=$True)]
+  [string] $repoName,
+  [Parameter(Mandatory=$True)]
+  [string] $Language
+)
+
```

Comment:
Can we try adding `Set-StrictMode -Version 3` here? Assuming your code paths through common follow this it will help identify any undefined variables or property usages.